lib/src/pdf_page_proxy.dart

```diff
@@ -35,9 +35,15 @@ class PDFPageProxy {
     _jsInternal.callMethod('cleanup', []);
   }
 
-  PageViewport getViewport(num scale, [int rotate = null]) {
-    JsObject jsViewport =
-        _jsInternal.callMethod('getViewport', [scale, rotate]);
+  PageViewport getViewport(num scale,
```

Comment:
Might be good to make these named parameters at this point to mirror the PDF.js API and to avoid positional bools.

Reply:
I did make rotation, dontFlip named parameters because they are optional. scale is required so I left it